src/systems/filecoin_mining/storage_mining/storage_miner_actor.id

```diff
@@ -82,18 +81,25 @@ type StorageMinerActorCode struct {
     // Can be filled in once InvokeMethod is implemented for this actor.
 }
 
+type MinerKeyChange struct {
+    NewKey       addr.Address
+    EffectiveAt  block.ChainEpoch
+}
+
 type MinerInfo struct {
     // Account that owns this miner.
     // - Income and returned collateral are paid to this address.
     // - This address is also allowed to change the worker address for the miner.
-    Owner                   address.Address  // Must be an ID-address.
+    Owner    addr.Address  // Must be an ID-address.
 
     // Worker account for this miner.
     // This will be the key that is used to sign blocks created by this miner, and
     // sign messages sent on behalf of this miner to commit sectors, submit PoSts, and
     // other day to day miner activities.
-    Worker                  address.Address
-    WorkerVRFKey            filcrypto.VRFPublicKey
+    _worker  addr.Address
+
+    _updateMinerPendingKeyChange(mkc MinerKeyChange)
+    GetWorker(epoch block.ChainEpoch) addr.Address
```

Comment:
The parameter to this method is unnecessary. 100% of the invocations pass the current epoch, the epoch of the state being inspected. There understandably seems to be no reason to want to know a miner's key some epoch in the future. 

I see further down that it has side-effects which I think we should strictly separate from access, and are ineffective in read-only contexts anyway. Given this I think the method can be removed and the member exported and consulted directly.